Xamarin.Forms.Core/Markup/BindableObjectExtensions.cs

```diff
@@ -0,0 +1,183 @@
+﻿using System;
+
+namespace Xamarin.Forms.Markup
+{
+	public static class BindableObjectExtensions
+	{
+		const string bindingContextPath = Binding.SelfPath;
+
+		/// <summary>Bind to a specified property</summary>
```

Comment:
we usually don't do inline doc

Reply:
Yes, I noticed; I did see some comments in existing Forms source with 'Note to documenters:'. Would that be more appropriate?